kubernetes/check-pods.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash -e
```

Comment:
Please rename the script to `wait-until-ready.sh`.

Reply:
Done.